Playgrounds/5a_flatMap_for_Array.playground/Contents.swift

```diff
@@ -243,19 +243,23 @@ Upon receiving an argument, a function is returned that takes the remaining para
 A new function is returned for every argument, until the final argument is supplied,
 then the whole expression will be evaluated and a value can be returned.
 
+Note that in Swift 3, currying `func` declaration syntax was removed from the language, therefore a more elaborate syntax is needed to achieve currying.
```

Comment:
added this hint text that currying `func` declaration syntax was removed from Swift - maybe some more explanation would help